app/src/main/res/values-sw600dp-land/dimens.xml

```diff
@@ -340,7 +340,7 @@
   <dimen name="pin_password_activity_padding_top">48dp</dimen>
   <dimen name="pin_password_activity_hello_text_view_margin_start">0dp</dimen>
   <dimen name="pin_password_activity_hello_text_view_margin_end">0dp</dimen>
-  <dimen name="pin_password_activity_input_with_icon_width">0dp</dimen>
+  <dimen name="pin_password_activity_input_with_icon_width">-1px</dimen>
```

Comment:
For this & the other files, please include a comment explaining what this means (since it's otherwise a bit of a code smell since negative width doesn't make sense).

Reply:
Added a comment, thanks